content/Foundation/Membership/LOGOS/secondstate.svg

```diff
@@ -1,71 +1,133 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 595.28 595.28" style="enable-background:new 0 0 595.28 595.28;" xml:space="preserve">
-<style type="text/css">
+
+<svg
+   version="1.1"
+   id="图层_1"
```

Comment:
Is everybody confident in using Unicode? ;)

Reply:
Good point, I removed that